charm/docs/tutorial/getting-started.md

```diff
@@ -1,31 +1,34 @@
+<!-- This tutorial should be updated in the src-docs directory. -->
+
 # Getting Started
```

Comment:
I would rename the tutorial to something like 
` # Deploy the NetBox charm for the first time`

"Getting started" implies more of a how-to guide than a tutorial, and also doesn't provide details about what the tutorial experience is going to be.